components/Ripple/src/MDCRippleTouchController.m

```diff
@@ -19,6 +19,14 @@ @implementation MDCRippleTouchController {
 }
 
 - (instancetype)initWithView:(UIView *)view {
+  self = [self init];
+  if (self) {
+    [self addRippleToView:view];
```

Comment:
Please don't call `addRippleToView:` from the initializers. Since the class may be subclassed, calling a public API on the object can result in incomplete initialization when called.  If this API (`initWithView:`) is absolutely necessary, please move the necessary implementation into a private method that isn't visible to subclasses.  Ideally, I think the necessary work would be in the `init` method directly.

Reply:
Done. Very good point and thanks for finding this